core/panic.c

```diff
@@ -28,6 +28,10 @@
 #include "panic.h"
 #include "arch/panic_arch.h"
 
+#if DEVELHELP && defined MODULE_PS
```

Comment:
`if DEVELHELP` is correct, (it may be defined, but defined as zero)
On May 15, 2015 8:46 AM, "Kaspar Schleiser" <notifications@github.com>
wrote:

> In core/panic.c
> <https://github.com/RIOT-OS/RIOT/pull/2950#discussion_r30388327>:
>
> > @@ -28,6 +28,10 @@
> >  #include "panic.h"
> >  #include "arch/panic_arch.h"
> >
> > +#if DEVELHELP && defined MODULE_PS
>
> defined(develhelp) && defined(module_ps) ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/2950/files#r30388327>.
>